src/meteodatalab/operators/lateral_operators.py

```diff
@@ -158,7 +158,7 @@ def fill_undef(field: xr.DataArray, radius: int, frac_val: float) -> xr.DataArra
         field.assign_coords(xy_coords)
         .rolling({"x": n, "y": n}, center=True)
         .construct(dims)
-        .sel(idx)
```

Comment:
was that not tested?

Reply:
It was part of a test but the previous values for the step coordinates were the same as the index